frontend/src/components/dashboard/account-components/SignedInNavItems.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+
+import {Link} from 'react-router-dom';
+
+import {
+  List, ListItem,
+  ListItemIcon, ListItemText,
+} from '@material-ui/core';
+
+import FavoriteIcon from '@material-ui/icons/Favorite';
+
+const useStyles = makeStyles((theme) => ({
+  link: {
+    textDecoration: 'none',
+    color: theme.palette.text.primary,
+  },
+}));
+
+export default function SignedInNavItems() {
```

Comment:
Will just have to be careful when merging with Nat's PRS since she has this component too